mappings/net/minecraft/client/gui/widget/layout/RotatableLayoutWidget.mapping

```diff
@@ -0,0 +1,86 @@
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual y
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will be its x coordinate.
+			ARG 1 widget
+		METHOD m_ltqylntt getRotatedHeight (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;)I
+			COMMENT Returns the height of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual height
+			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL}, it will be
+			COMMENT its width.
+			ARG 1 widget
+		METHOD m_ntoedljy getRotatedWidth (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;)I
+			COMMENT Returns the width of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual width
+			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL}, it will be
+			COMMENT its height.
+			ARG 1 widget
+		METHOD m_otbzsyab getRotatedHeight (Lnet/minecraft/unmapped/C_dpukplka;)I
+			COMMENT Returns the height of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual height
+			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL}, it will be
+			COMMENT its width.
+			ARG 1 widget
+		METHOD m_sfolbtqa getRotatedX (Lnet/minecraft/unmapped/C_dpukplka;)I
+			COMMENT Returns the "x" coordinate of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual x
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will be its y coordinate.
+			ARG 1 widget
+		METHOD m_whgwfqwu setRotatedX (Lnet/minecraft/unmapped/C_zuxsitrm$C_isrpishr;I)V
+			COMMENT Sets the "x" coordinate of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will set the actual x
+			COMMENT coordinate of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL},
+			COMMENT it will set its y coordinate.
+			ARG 1 widget
+			ARG 2 x
+		METHOD m_wkaarfup getRotatedWidth (Lnet/minecraft/unmapped/C_dpukplka;)I
+			COMMENT Returns the width of the given {@code widget} rotated with this orientation.
+			COMMENT <p>
+			COMMENT If the orientation is {@link Orientation#HORIZONTAL HORIZONTAL}, this will be the actual width
+			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL VERTICAL}, it will be
```

Comment:
```suggestion
			COMMENT If the orientation is {@link Orientation#HORIZONTAL}, this will be the actual width
			COMMENT of the {@code widget}. Otherwise, if it is {@link Orientation#VERTICAL}, it will be
```